packages/aws-cdk/lib/cli-arguments.ts

```diff
@@ -0,0 +1,1253 @@
+// -------------------------------------------------------------------------------------------
+// GENERATED FROM packages/aws-cdk/lib/config.ts.
+// Do not edit by hand; all changes will be overwritten at build time from the config file.
+// -------------------------------------------------------------------------------------------
+/* eslint-disable @typescript-eslint/comma-dangle, comma-spacing, max-len, quotes, quote-props */
+/**
+ * The structure of the CLI configuration, generated from packages/aws-cdk/lib/config.ts
+ *
+ * @struct
+ */
+export interface CliArguments {
+  /**
+   * The CLI command name followed by any properties of the command
+   */
+  readonly _: Array<string>;
+
+  /**
+   * Global options available to all CLI commands
+   */
+  readonly globalOptions?: GlobalOptions;
```

Comment:
`globalOptions` is its own property bag because some global options have the same name as a command (`notices`, for example). This is a departure from what we receive from the input, but I think this is a cleaner interface.

Reply:
This make sense. The other option would be that each command interface is extending `GlobalOptions` and we don't have a separate GlobalOptions property bag.